### ISSUE TITLE
Mention that network=host does not work on Docker Desktop for Linux

### ISSUE THREAD

**Issue by av-jeroen** [docker/docs#17014]
File: [network/host.md](https://docs.docker.com/network/host/)

Mention that network=host does not work on Docker Desktop for Linux

**Comment by Satyxm** [comment]
Hey! I would like to work on this issue. I am a first-time contributor to Docker, and as far as I am understanding this issue,  is that I have to add the line: 
The host networking driver only works on Linux hosts, and is not supported on Docker Desktop for Mac, Docker Desktop for Windows, Docker EE for Windows Server, or Docker Desktop for Linux. 

Instead of: 
The host networking driver only works on Linux hosts, and is not supported on Docker Desktop for Mac, Docker Desktop for Windows, or Docker EE for Windows Server.

**Comment by thaJeztah** [comment]
@dvdksn @akerouanton was this ticket addressed in one of the rewrites, or should it be reopened?

**Comment by dvdksn** [comment]
@thaJeztah no, I haven't moved forward with this. We can reopen, would like to get @akerouanton blessing on what to do 🙏🏻

**Comment by dvdksn** [comment]
Fixed. Docs now document that it works with docker desktop + Desktop limitations (layer 4 only, opt-in since v4.34).
